h/stcbackground.h

```diff
@@ -0,0 +1,54 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which 
+  accompanies this distribution, and is available at 
+  https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+#ifndef __STCBACKGROUND__
+#define __STCBACKGROUND__
+
+#include "stcbase.h"
+#define STC_BG_CB_LABEL_LEN 30
```

Comment:
Same as the structures below, this doesn't belong here,

Reply:
i have moved one structure, but as stccallbackdata is part of callback signature, so kept that in .h

we are exposing   struct STCIntervalCallbackData_tag* callbackData, 
so kept the definition in .h and related constant
```
typedef int (*STCIntervalCallback)(STCBase* stcBase, 
                                    STCModule* module, 
                                    struct STCIntervalCallbackData_tag* callbackData, 
                                    void* userData
                                   );
```

Reply:
moved struct definition to .c